src/gt4py/next/program_processors/runners/dace_iterator/workflow.py

```diff
@@ -115,15 +116,39 @@ class Meta:
         model = DaCeTranslator
 
 
+class CompiledDaceProgram(stages.CompiledProgram):
+    sdfg_program: dace.CompiledSDFG
+    sdfg_arg_position: list[
+        int
+    ]  # map SDFG argument to its position in program ABI, -1 if argument is not used
```

Comment:
I try to never mention formatting and I follow the doctrine "the output of ruff is final", but consider to put the comment on its own line as it looks awful.